src/App.css

```diff
@@ -28,49 +28,315 @@
   color: #61dafb;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
+body {
+  background-color: #f5f6fa;
+}
 
-  to {
-    transform: rotate(360deg);
-  }
+.root {
+  background-color: #fff;
+  margin: 6vw;
+  box-shadow: 50px 50px 20px #afabab36;
 }
 
-.booklist,
-.bookform {
-  width: 70vw;
-  margin: auto;
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 6.25rem;
+}
+
+nav .logo {
+  display: flex;
+  width: 35%;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo h1 {
+  color: #4e97ff;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.875rem;
+  font-weight: bolder;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.logo h3,
+.logo h4 {
+  font-family: Montserrat, sans-serif;
+  font-size: 0.813rem;
+  font-weight: bolder;
+  font-style: normal;
+  color: rgb(70, 68, 68);
+}
+
+.logo h4 {
+  color: rgba(70, 68, 68, 0.425);
+}
+
+nav i {
+  padding: 0.875rem;
+  border: solid 1px #e8e8e8;
+  border-radius: 50%;
+  color: #4e97ff;
+}
+
+main {
+  background-color: #fafafa;
+  padding: 0 6.25rem;
+  padding-top: 20px;
+  padding-bottom: 150px;
+}
+
+.booklist {
   display: flex;
   flex-direction: column;
   justify-content: center;
+}
+
+tr {
+  height: 10.625rem;
+  border-radius: 4px;
+  border: solid 1px white;
+  margin-bottom: 50px;
+}
+
+.eachbook {
+  width: inherit;
+  display: flex;
+  justify-content: start;
   align-items: center;
-  margin-bottom: 70px;
+  background-color: white;
+  padding: 1vw;
+  padding-right: 50px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  margin-bottom: 0.983rem;
 }
 
-.booklist tr {
-  text-align: center;
-  height: 45px;
+.div1 {
+  width: 50%;
+}
+
+.category {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
```

Comment:
These rules appear to be repeated several times. I think it would be better if you refactored them into a single rule. This would make your CSS more maintainable. ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)